src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -78,15 +79,31 @@ class PackageDetailTab extends mixin(StoreMixin) {
     });
   }
 
-  componentDidMount() {
-    super.componentDidMount(...arguments);
-
+  retrievePackageInfo() {
```

Comment:
instead of having the same methods being called in `componentDidMount` and `componentDidUpdate` I consolidated the methods under `retrievePackageInfo`